video/clip.go

```diff
@@ -10,9 +10,10 @@ import (
 )
 
 type ClipStrategy struct {
-	Enabled   bool
-	StartTime float64 `json:"start_time,omitempty"`
-	EndTime   float64 `json:"end_time,omitempty"`
+	Enabled    bool
+	StartTime  float64 `json:"start_time,omitempty"`
+	EndTime    float64 `json:"end_time,omitempty"`
+	PlaybackId string  `json:"playback_id,omitempty"` // playback-id of asset to clip
```

Comment:
Petty, but `ID` should be capitalised in the variable name to follow Golang guidelines 

```suggestion
	PlaybackID string  `json:"playback_id,omitempty"` // playback-id of asset to clip
```

Reply:
Fixed